### PR TITLE
Upgrade to version 3.4.1

### PR DESCRIPTION
**Pull request by rampr**

### CODE REVIEW

File: android/OnesignalPlugin.java [hunk 1]
```diff
@@ -198,15 +198,15 @@ public static JSONObject get(OSNotification notification) {
...
+        os_data.put("notification_id", payload.notificationID);
         os_data.put("is_foreground", notification.isAppInFocus);
         os_data.put("title", payload.title);
         os_data.put("body", payload.body);
         additional_data = payload.additionalData;
         if (additional_data != null) {
           os_data.put("additional_data", additional_data.toString());
         }
-        os_data.put("launch_url", payload.launchUrl);
+        os_data.put("launch_url", payload.launchURL);
```
Comment (rampr) [inline android/OnesignalPlugin.java:209]: the params have changed to match with the iOS version

File: android/manifest.xml [hunk 2]
```diff
@@ -34,27 +34,27 @@
...
+  <intent-filter>
+    <action android:name="android.intent.action.BOOT_COMPLETED" />
+    <action android:name="android.intent.action.QUICKBOOT_POWERON" />
+  </intent-filter>
+</receiver>
+<receiver android:name="com.onesignal.UpgradeReceiver" >
+  <intent-filter>
+    <action android:name="android.intent.action.MY_PACKAGE_REPLACED" />
+  </intent-filter>
+</receiver>
```
Comment (rampr) [inline android/manifest.xml:59]: only fixed indentation

File: android/OnesignalPlugin.java (hunk 1 above)
Comment (bijoshtj) [reply android/OnesignalPlugin.java:209]: dont we need to change game code ? or this was a bug?

Comment (rampr) [reply android/OnesignalPlugin.java:209]: game code doesn't need to change. the onesignal android sdk has updated their API in the latest versions to match with the iOS versions.

Older versions of the SDK were using launchUrl, newer versions would use launchURL

**Review by sksethi25 — APPROVED** [review]

**Review by bijoshtj — APPROVED** [review]

**Review by rohanhc — APPROVED** [review]

**Pull request merged**